src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -520,6 +520,16 @@ export class IntegrationListDirectory extends AsyncComponent<
                   {tct('No Integrations found for "[searchTerm]".', {
                     searchTerm: searchInput,
                   })}
+                  <EmptyResultsBodyBold>
+                    {t("Not seeing what you're looking for?")}
+                  </EmptyResultsBodyBold>
+                  <EmptyResultsBody>
+                    {tct('[link:Build it on the Sentry Integration Platform.]', {
```

Comment:
@ceorourke I don't think we need this `<EmptyResultsBody>` because it will inherit the styles from the `<EmptyResultsBody>` above it.

Reply:
@ceorourke also by putting everything under the `<EmptyResultsBody>`, I think we looks the center alignment that we get from `<EmptyResultsContainer>`